framer/Material-UI.framerfx/code/Typography.tsx

```diff
@@ -29,7 +29,6 @@ interface Props {
     | 'caption'
     | 'button'
     | 'overline'
```

Comment:
Are these changes ok?

Reply:
@mbrookes the `inherit` value looks weird.

Reply:
My bad, it's a valid value.